setup.py

```diff
@@ -5,13 +5,13 @@
 
 install_requires = [
     "geopandas",
+    "numpy<1.21,>=1.17",
```

Comment:
Why `<1.21`? If possible, we should not restring the maximum version, only the minimum.

Reply:
The latest version of numba (v.0.54.0) specifies [numpy<1.21,>=1.17](https://pypi.org/project/numba/). Alternatively, we could use an earlier version of numba? v.0.53.1 specifies [NumPy >=1.15 (can build with 1.11 for ABI compatibility)](https://pypi.org/project/numba/0.53.1/)

Reply:
If numba needs certain versions let's leave it to them to take care of pins. We just list `numba`.

Reply:
`numba` was throwing a dependency error when trying to install Dask-GeoPandas.